### DISCUSSION TITLE
Subscribe to custom issue notifications

### DISCUSSION THREAD

**Discussion by swrobel** [community/community#62709] [Mobile]
Customizable notifications for issues and PRs is one of the best features rolled out in years on the web. The mobile app needs it too. 
<img width="481" height="444" alt="Screenshot 2026-01-29 at 8 34 19 AM" src="https://github.com/user-attachments/assets/c908f6f5-3d9f-4131-96ee-a0bedf553b3d" />

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.124.0 (105848740)
Device: iPad Mini 6 (iOS 17.0)
</details>

**Comment by mattruggio** [comment]
Hey @swrobel thanks for the heads up! I want to make sure I am understanding the request here a bit more: it sounds like what you are referring to is the lack of granularity when controlling subscriptions via the mobile app. For example the web would show this customizable UI element:

<img width="441" alt="image" src="https://github.com/community/community/assets/906763/c925ee56-dfd4-4daa-b3ea-8fc443b9fc63">

While on mobile, it only limits you to a more binary subscribe/unsubscribe (mobile is missing the custom section):

<img width="276" alt="image" src="https://github.com/community/community/assets/906763/5177529d-b5a9-40e5-80e8-b0e5f7e643f4">

Does this sound like a fair explanation of this specific issue?

**Reply by swrobel** [reply]
> it sounds like what you are referring to is the lack of granularity when controlling subscriptions via the mobile app. For example the web would show this customizable UI element:
> <img alt="image" width="441" src="https://private-user-images.githubusercontent.com/906763/259447124-c925ee56-dfd4-4daa-b3ea-8fc443b9fc63.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk3MDQ4MzEsIm5iZiI6MTc2OTcwNDUzMSwicGF0aCI6Ii85MDY3NjMvMjU5NDQ3MTI0LWM5MjVlZTU2LWRmZDQtNGRhYS1iM2VhLThmYzQ0M2I5ZmM2My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDEyOVQxNjM1MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hM2NlNTU1YWVlNjI0NTM5YzFlMmNjNDUwNmQwZjY5ZGM3YThjYzFlY2VmNzc3MTlhN2M5ZWJmYWI2NTNlMmM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.p64ddDcYJsAAIgK83rLK9FLDAQMd0k3i9eOnqrX8_I8">

You are absolutely correct in your explanation. Has there been any progress on this issue? This is a major functionality gap.

**Comment by simssem** [comment]
M
